WcaOnRails/spec/controllers/registrations_controller_spec.rb

```diff
@@ -169,16 +167,29 @@
       expect(registration.competition_id).to eq competition.id
     end
 
+    it "can re-create registration after it was deleted" do
+      registration = FactoryGirl.create :registration, :accepted, :deleted, competition: competition, user_id: user.id
```

Comment:
It's disconcerting to me that it looks like we're creating a registration that is both accepted and deleted.

Could you split this up into a couple more steps? First create an accepted registration, then delete it, and then recreate it?